DESCRIPTION

```diff
@@ -20,7 +20,9 @@ Suggests:
     knitr,
     rmarkdown,
     testthat,
-    ggthemes
+    ggthemes,
+    ggrastr,
```

Comment:
Would you consider replacing ggrastr with [scattermore](https://github.com/exaexa/scattermore)?

Reply:
Seems reasonable, though I didn't find any test or vignette coverage on this code path, so I'm hesitant to do anything that's not totally transparent.

Reply:
good point, such a change would deserve a test — maybe a different pull request